src/main/java/handlers/InformationHandler.java

```diff
@@ -0,0 +1,39 @@
+package handlers;
+
+import java.util.LinkedList;
+import java.util.Queue;
+
+/**
+ * Handler for the information to be shown.
+ */
+public class InformationHandler {
+
+    private Queue<String> queue = new LinkedList<>();
+
+    /**
+     * Add information to the queue.
+     * @param info The new information.
+     */
+    public void addInformation(final String info) {
+        queue.add(info);
+    }
+
+    /**
+     * Get information from the queue and remove if it exists, otherwise returns empty.
+     * @return The information.
```

Comment:
Again capital letters. Also the information is maybe kind of vague. Might be better to say something like "string with the current information"

Reply:
information about events happening*